File: project_euler/005 - Smallest multiple/5.py
import sys

def main():

    for i in range(20, sys.maxsize):

        evenly_divisible = True

        for j in range(20):
            if not i % (j+1) == 0:
                evenly_divisible = False
                break

        if evenly_divisible:
            print(f"Smallest positive number that is evenly divisible by all of the numbers from 1 to 20: {i}")
            break


if __name__ == "__main__":
    main()

File: project_euler/005 - Smallest multiple/5.c
#include<stdio.h>
main(){
    int i, c;

    for(i=2520;;i++){

        if ((i%11)==0 && (i%12)==0 && (i%13)==0 && (i%14)==0 && (i%15)==0 && (i%16)==0 && (i%17)==0 && (i%18)==0 && (i%19)==0 && (i%20)==0){
            printf("The smallest number divisible by all numbers 1-20 is : %d\n", i);
            break;
    }
}
}

